examples/experimental/dagster-airlift/dagster_airlift/constants.py

```diff
@@ -3,4 +3,6 @@
 TASK_MAPPING_METADATA_KEY = "dagster-airlift/task_mapping"
 # This represents the timestamp used in ordering the materializatons.
 EFFECTIVE_TIMESTAMP_METADATA_KEY = "dagster-airlift/effective_timestamp"
-DAG_RUN_ID_TAG_KEY = "dagster-airlift/dag-run-id"
+DAG_RUN_ID_TAG_KEY = "dagster-airlift/airflow-dag-run-id"
```

Comment:
should we have an additional prefix for "from airflow"

Reply:
well i thought the `airflow` prefix in front of `dag-run-id` et all implied that.

I think the pertinent information might be that it is the proxy that inserted it